modules/dictionary/php/categories.class.inc

```diff
@@ -47,7 +47,7 @@ class Categories extends \NDB_Page
                 'candidate_parameter_view',
                 'candidate_parameter_edit'
             ]
-        ) || !$user->hasPermission('superuser')
```

Comment:
Why is this if statement here at all? Why is candidate_parameter_* being treated specially in the dictionary?